flight/Flight.php

```diff
@@ -104,56 +101,53 @@ private function __clone()
 
     /**
      * Registers a class to a framework method.
-     * @template T of object
-     * @param  string $name Static method name
+     *
+     * # Usage example:
      * ```
      * Flight::register('user', User::class);
      *
      * Flight::user(); # <- Return a User instance
      * ```
-     * @param  class-string<T> $class Fully Qualified Class Name
-     * @param  array<int, mixed>  $params   Class constructor params
-     * @param  ?Closure(T $instance): void $callback Perform actions with the instance
-     * @return void
+     *
+     * @param string $name Static method name
+     * @param class-string<T> $class Fully Qualified Class Name
+     * @param array<int, mixed>  $params   Class constructor params
+     * @param ?Closure(T $instance): void $callback Perform actions with the instance
+     *
+     * @template T of object
      */
-    public static function register($name, $class, $params = [], $callback = null)
+    public static function register($name, $class, $params = [], $callback = null): void
     {
-        static::__callStatic('register', func_get_args());
+        static::__callStatic('register', [$name, $class, $params, $callback]);
```

Comment:
I've wondered if these 2 methods even need to exist.....cause __callStatic will just call them anyways.....seems weird.

Reply:
I did it like this because of a phpcs rule about unused parameters